client/app/dedicated/server/working-status/dedicated-server-working-status.routes.js

```diff
@@ -0,0 +1,7 @@
+angular.module("App").config(($stateProvider) => {
+    $stateProvider.state("app.dedicated.server.working-status", {
+        url: "/travaux",
+        templateUrl: "dedicated/server/working-status/dedicated-server-working-status.html",
+        controller: "WorkingStatusCtrl"
```

Comment:
You could have set the `controllerAs` here, but once again, using a component would be better :)